pages/common/bfg.md

```diff
@@ -0,0 +1,13 @@
+# bfg
+
+> Remove large files or passwords from Git history like git-filter-branch.
+> Note: if your repository is connected to a remote, you will need to force push to it.
+> More information: <https://rtyley.github.io/bfg-repo-cleaner/>.
+
+- Remove a file with sensitive data and leave the latest commit untouched:
```

Comment:
"And" makes it sound as though it's doing an action, but it's really just an omission.
```suggestion
- Remove a file with sensitive data but leave the latest commit untouched:
```

Reply:
I agree!